### PR TITLE
Improve not logged in -implementatation

### PR DESCRIPTION
**Pull request by janimattiellonen**
axios now wrapped in Client for easier configuration (less configuration in multipla places)
added 401 listener to axios so that we can redirect to a login page
added a new path listener to django that will direct requests to certain path(s) to our React app
ConfirmDialog now acceppts custom confirmation button title
added translations for login page content

### CODE REVIEW

File: omahelsinki/urls.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
     path('', include('helusers.urls')),
     path('mydata/', TemplateView.as_view(template_name='react_base.html'), name='mydata'),
     path('welcome/', TemplateView.as_view(template_name='react_base.html'), name='welcome'),
+    re_path(r'^app/', TemplateView.as_view(template_name='react_base.html'), None, name='app'),
```
Comment (igordavydsson) [inline omahelsinki/urls.py]: What's up with `None` there? Why do you need it?

Comment (janimattiellonen) [reply omahelsinki/urls.py]: ah, so used to php so I thought you had to provided a default value if you omitted on.
The parameters seems to be named so I aguess you can just omit it?

`re_path(r'^app/', TemplateView.as_view(template_name='react_base.html'), name='app')` ?

Comment (igordavydsson) [reply omahelsinki/urls.py]: yes, you can omit "non-named" parameters :)

**Comment by igordavydsson** [comment]
Could you squash your commits? This second commit does not make sense, if you can just not have that `None~ in there in the first place.

**Comment by janimattiellonen** [comment]
better?

**Pull request merged**